apps/src/aichat/views/ChatEventView.tsx

```diff
@@ -40,11 +41,18 @@ function formatModelUpdateText(update: ModelUpdate): string {
     )?.name;
   }
 
-  const updatedText = updatedToText
-    ? `has been updated to ${updatedToText}.`
-    : 'has been updated.';
-
-  return `${fieldLabel} ${updatedText} ${timestampToLocalTime(timestamp)}`;
+  const modelUpdateText = updatedToText
+    ? aichatI18n.modelUpdateText({
+        fieldLabel: fieldLabel,
+        updatedToText: updatedToText.toString(),
```

Comment:
I think this key should be `updatedText`. See https://github.com/code-dot-org/code-dot-org/blob/bd6ff1fe8c561ede09badb32f568f57fc405cd49/apps/i18n/aichat/en_us.json#L65. Currently it's `undefined`.

Reply:
![Screenshot 2024-12-13 at 4 36 31 PM](https://github.com/user-attachments/assets/b7699bff-79bc-42cc-9734-91d9239ca451)

Reply:
I see you merged - maybe do a fast follow-up?

Reply:
Yeah, gonna do a fast follow

Reply:
I created a fast-follow PR for this here: https://github.com/code-dot-org/code-dot-org/pull/62955

Thanks for catching that!